.github/workflows/scripts/test/test_check_python_deps.py

```
@@ -0,0 +1,152 @@
+        """Should extract base package name from extras syntax."""
+        with patch("importlib.metadata.distributions", return_value=[]):
+            with patch.object(
+                check_python_deps,
+                "REQUIREMENTS_FILE",
+                unittest.mock.MagicMock(
+                    exists=lambda: True,
+                    read_text=lambda: "requests[security]>=2.0\n",
+                ),
+            ):
+                result = check_python_deps.get_missing_dependencies()
+                self.assertIn("requests", result)
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)